### DISCUSSION TITLE
Team Health Test

### DISCUSSION THREAD

**Discussion by scrowlan** [scrowlan/tradejournal#20] [General]
### Description: 
Team Health refers to the set of metrics we collect to determine how healthy a team is. other stuff about them too

The following metrics are currently active:

- Churn on in progress work
- Cycle time

**Comment by scrowlan** [comment]
# Churn on in progress work

## Actively being tracked

## Goal
80% of teams achieving less than 20% churn on in progress work

## Description:
The higher churn is the less predictable we can be.

**Reply by scrowlan** [reply]
Comment about this one

**Reply by LizzySoltis** [reply]
test test test

**Comment by scrowlan** [comment]
# Burn out

## Not actively being tracked

## Goal
undetermined

## Description
We don't want burn out

**Reply by scrowlan** [reply]
Reply here

**Reply by scrowlan** [reply]
> Reply here

quote replying here

**Comment by scrowlan** [comment]
# Cycle time on tasks

## Actively being tracked

## Goal
80% of tasks closed in less than 5 days

## Description
The more consistent our cycle time is the better we get at being predictable. Also, the smaller we can make our tasks the easier it is to clearly define the scope and value being delivered in each task.
